.github/workflows/nix.yml

```diff
@@ -63,3 +63,9 @@ jobs:
 
       - name: nix build
         run: nix run github:nixos/nixpkgs/nixos-23.11#nixci
+
+      - name: integration check
+        run: |
+          nix build --accept-flake-config -L .#nixsgx-test-sgx-azure
+          docker load -i result
+          docker run -i --env GRAMINE_DIRECT=1 --privileged --init --rm nixsgx-test-sgx-azure:latest | grep -q -F 'Hello, world!'
```

Comment:
Nit: is `--privileged` needed if we don't use SGX-specifcic device files from `/dev/` in direct mode? 🤔

Reply:
entweder privilegiert oder mit einem benutzerdefinierten seccomp-Profil... see https://gramine.readthedocs.io/en/stable/installation.html